tool/state-processor/src/balances/mod.rs

```diff
@@ -35,12 +35,9 @@ impl<S> Processor<S> {
 			.take_value(b"Balances", b"TotalIssuance", "", &mut para_ring_total_issuance)
 			.take_map(b"Balances", b"Locks", &mut para_ring_locks, get_hashed_key);
 
-		log::info!("check solo ring locks, there should not be any `solo_ring_locks`");
-		check_locks(solo_ring_locks);
-		log::info!("check solo kton locks, there should not be any `solo_kton_locks`");
-		check_locks(solo_kton_locks);
-		log::info!("check para locks, there should not be any `para_ring_locks`");
-		check_locks(para_ring_locks);
+		if self.para_state.exists(b"Balances", b"Locks") {
+			log::error!("check para `Balances::Locks`, it isn't empty");
+		}
```

Comment:
The old version checks both solo locks and para locks, the new version only checks the para locks. Is this expected?

Reply:
Those have already been checked in the process step.

But we skip the para locks process.